CognitiveConcierge-iOS/CognitiveConcierge/EntertainmentOptionsViewController.swift

```diff
@@ -47,6 +47,8 @@ class EntertainmentOptionsViewController: UIViewController {
     private let kInternetAlertMessage = "Make sure your device is connected to the internet."
     private let kCredentialsAlertTitle = "Conversation Service Unavailable"
     private let kCredentialsAlertMessage = "Please make sure you entered your Conversation service credentials correctly."
+    private let kRestaurantsAlertTitle = "Please Note"
+    private let kRestaurantsAlertMessage = "This is currently a demo app for only Restaurants. Feel free to pull and add the functionality for other tiles."
```

Comment:
"This is a demo application for restaurant suggestions only.  Feel free to pull the open source code and add functionality for other tiles"  What about that?